androidbrowserhelper/CHANGELOG.md

```diff
@@ -1,3 +1,18 @@
+## 2.6.2
+
+* The TWA launcher will no longer wait for the animation to finish before launching the Custom
+Tabs Activity, which improves launch time. This change does not affect the user-visible launch
+animation. The PR for the change is
+[#520](https://github.com/GoogleChrome/android-browser-helper/pull/520).
+This behavior can be reverted by add following lines in TWA app's AndroidManifest.xml inside
```

Comment:
nit: probably add a new line before this

Reply:
Done.